react/fields/Autosuggest/Autosuggest.less

```diff
@@ -0,0 +1,55 @@
+@import (reference) "~seek-style-guide/theme";
+
+.container {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+}
+
+.suggestionsContainer {
+  display: none;
+
+  .containerOpen & {
+    z-index: 1;
+    display: block;
+    position: absolute;
+    background-color: @sk-white;
+    box-shadow: 0 9px 30px rgba(0, 0, 0, 0.4);
+    padding: @grid-row-height 0;
+    border-radius: 2px;
+    min-width: 100%;
+    max-width: 100%;
+    max-height: 385px;
+    overflow-y: auto;
+    top: @grid-row-height * 6;
+
+    @media only screen and (min-width: 920px) {
+      max-width: initial;
+    }
+  }
+}
+
+.sectionTitle {
+  .rawText();
+
+  line-height: (@grid-row-height * 4);
+  height: (@grid-row-height * 4);
+  margin: @grid-row-height (@grid-gutter-width * 0.75) 0;
+  border-top: 1px dashed @sk-mid-gray-light;
+  color: @sk-mid-gray-dark;
+}
+
+.suggestion {
+  .rawText();
+
+  line-height: (@grid-row-height * 4);
+  height: (@grid-row-height * 4);
+  padding: 0 (@grid-gutter-width * 0.75);
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.suggestionFocused {
+  background-color: @sk-gray-light;
```

Comment:
@tcrook1 also our focus/hover colour for suggestion differs, should it be `@sk-mid-gray-light` (#d6d6d6) or `@sk-gray-light` (#eeeeee)